src/AppBundle/Controller/RfXAnalysisController.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * This file contains the code that powers the RfX Analysis page of xTools.
+ *
+ * @category RfXAnalysis
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfa
+ */
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Xtools\ProjectRepository;
+use Xtools\RFA;
+
+/**
+ * Class RfXAnalysisController
+ *
+ * @category RfXAnalysis
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfa
```

Comment:
Is it required by Travis? I don't see it being used in the docs for other files. `@author` to me doesn't make sense, especially since we're only putting in "XTools Team". Contact info is naturally found on the README, just like the license is in LICENSE, etc. My two cents, but if we put this here we should keep it consistent and repeat it across all the files.